addons/autocomplete_googleplaces/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8" ?>
+
+<odoo>
+
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.autocomplete.googleplaces</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="website.res_config_settings_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[@id='autocomplete_googleplaces_setting']/div[hasclass('o_setting_right_pane')]" position="inside">
+                <div>
+                    <div class="content-group row mt16">
+                        <label class="col-4" for="google_places_api_key" string="API Key"/>
+                        <field class="col-6" name="google_places_api_key"/>
+                    </div>
+                    <div class="mt8">
+                        <a target="_blank" href="https://console.cloud.google.com/getting-started">
+                            <i class="fa fa-arrow-right"/>
+                            Create a Google Project and get a key
+                        </a>
+                        <a target="_blank" href="https://console.cloud.google.com/billing">
+                            <i class="fa fa-arrow-right"/>
+                            Enable billing on your Google Project
+                        </a>
```

Comment:
It seems weird to me that we have a link to "Enable billing", I believe it is ok to let customers take care of it when they create their project + key.
To check with the PO.

Reply:
I think its really not that obvious when you are creating a project on google cloud. I had to struggle a bit before understanding that I had to enable billing, because you don't get any call to action or even a warning, The warnings only come when making requests after that.

Also, this was specifically a request from PDE, as the other feature using a google api uses the same layout :)

Reply:
Alright.